### PR TITLE
Update Pulse.schelp

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**

### CODE REVIEW

File: HelpSource/Classes/Pulse.schelp [hunk 1]
```diff
@@ -8,10 +8,12 @@ Description::
 
 Band limited pulse wave generator with pulse width modulation.
 
+one thing you can try to do with supercollider pulse ugen: is
+creating several duplicate copies at a really low frequency, having a macro variable, to interpolate and detune a slickly little bit these frequencies, and fool around with pulse with modulation. when you apply these kinds of things, another really really cool thing you can do is layering several similar ugens with random variables, and different properties, in the exact same way you would do with  additive synthesis, or additive side chaining in the context of a sound mix.
 
 classmethods::
 
-method::ar, kr
```
Comment (snappizz) [inline HelpSource/Classes/Pulse.schelp]: Why was this removed?

File: HelpSource/Classes/Pulse.schelp [hunk 2]
```diff
@@ -8,10 +8,12 @@ Description::
 
 Band limited pulse wave generator with pulse width modulation.
 
+one thing you can try to do with supercollider pulse ugen: is
+creating several duplicate copies at a really low frequency, having a macro variable, to interpolate and detune a slickly little bit these frequencies, and fool around with pulse with modulation. when you apply these kinds of things, another really really cool thing you can do is layering several similar ugens with random variables, and different properties, in the exact same way you would do with  additive synthesis, or additive side chaining in the context of a sound mix.
```
Comment (telephon) [inline HelpSource/Classes/Pulse.schelp]: this text is better in a tutorial or sccode.org. Please keep the helpfiles fairly neutral.

**Comment by tiagmoraismorgado** [comment]
agreed telephon. i think you are absolutely right
